src/App.js

```diff
@@ -0,0 +1,31 @@
+import './App.css';
```

Comment:
put this in  a separate index.js file then refer that file here that will  make ur code cleaner